apps/prairielearn/assets/scripts/instructorAssessmentQuestionsClient.ts

```diff
@@ -12,13 +82,562 @@ onDocumentReady(() => {
     });
   });
 
-  $('[data-toggle="popover"]').popover({ sanitize: false });
+  function refreshTable() {
+    morphdom(
+      assessmentQuestionsTable as Node,
+      EditAssessmentQuestionsTable({
+        zones,
+        assessmentType,
+        showAdvanceScorePercCol,
+      }).toString(),
+    );
+  }
 
-  $('.js-sync-popover[data-toggle="popover"]')
-    .popover({ sanitize: false })
-    .on('show.bs.popover', function () {
-      $($(this).data('bs.popover').getTipElement()).css('max-width', '80%');
+  enableEditButton?.addEventListener('click', () => {
+    enableEditButton.style.display = 'none';
+    editModeButtons?.style.removeProperty('display');
+    refreshTable();
+  });
+
+  function renumberQuestions() {
+    let questionNumber = 1;
+    zones.forEach((zone: AssessmentQuestionZone) => {
+      zone.questions.forEach((question) => {
+        question.number = questionNumber.toString();
+        question.alternative_group_number = questionNumber;
+        if (question.is_alternative_group) {
+          let alternativeNumber = 1;
+          question.alternatives.forEach((alternative) => {
+            alternative.alternative_group_number = questionNumber;
+            alternative.number_in_alternative_group = alternativeNumber;
+            alternativeNumber += 1;
+          });
+        }
+        questionNumber += 1;
+      });
     });
+  }
 
-  document.querySelectorAll<HTMLElement>('.js-histmini').forEach((element) => histmini(element));
+  function swapQuestions(zoneNumber: number, questionNumber: number, targetQuestionNumber: number) {
+    const question = zones[zoneNumber].questions[questionNumber];
+    zones[zoneNumber].questions[questionNumber] = zones[zoneNumber].questions[targetQuestionNumber];
+    zones[zoneNumber].questions[targetQuestionNumber] = question;
+    renumberQuestions();
+    refreshTable();
+  }
+
+  on('click', '.zone-up-arrow-button', (e) => {
+    const zoneNumber = parseInt(
+      (e.target as HTMLElement).closest('button')?.dataset.zoneNumber ?? '0',
```

Comment:
Updated these (where applicable) to use "index" instead of number in [3c6791f](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/3c6791f5b7ef00282d313997acf9af8418da3660).